db/version_set.cc

```diff
@@ -1124,6 +1124,9 @@ VersionSet::Finalize(Version* v)
             else
                 elapsed_micros=0;
 
+            // reevaluating timed grooming ... seems to crush caching
+            elapsed_micros=0;
```

Comment:
Would be good to have a comment here stating that this explicitly and intentionally disables the timed grooming feature